tools/container/runtime/docker/docker.go

```diff
@@ -55,6 +55,10 @@ func Setup(c *cli.Context, o *container.Options) error {
 		return fmt.Errorf("unable to configure docker: %v", err)
 	}
 
+	if o.RuntimeEnableCDI {
+		cfg.Set("features", map[string]bool{"cdi": true})
+	}
+
```

Comment:
Note that we're already enabling CDI in the `Configure` call, so there is no need to do it here too.
```suggestion
```